src/ol/format/filter/logicalnary.js

```diff
@@ -0,0 +1,28 @@
+goog.provide('ol.format.filter.LogicalNary');
```

Comment:
`ol.format.filter.Not` and `ol.format.filter.LogicalNary` both provide an implementation for `ol.format.filter.Logical`. But I don't have a strong opinion on this, I removed the class.